Makefile

```diff
@@ -95,7 +95,7 @@ selfcheck: ## check that the Makefile is well-formed
 	@echo "The Makefile is well-formed."
 
 docker_build:
-	docker-compose build
+	docker build . -t openedx/xblock-sdk:latest
```

Comment:
usually we also tag the image with commit id

Reply:
In the build step most edx workflows only tag the `latest` commit and only tag with commit id before pushing the image.

Reply:
reverted this change.